assets/scss/admin-global.scss

```
@@ -7,3 +7,32 @@
 	margin-left: 3px;
 	vertical-align: super;
 }
+/*edit post page styles*/
```

Stylelint (17.11.1)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)